### ISSUE TITLE
[datepicker.MaterialCalendarGridView] crash on select a date range across two months (second month with 6 weeks)

### ISSUE THREAD

**Issue by fotospiro** [material-components/material-components-android#2285]
**Description:** 
It still crashes with **com.google.android.material:material:1.4.0**
See #2195 (updated)

**Comment by fotospiro** [comment]
I finally found the problem: `@style/` was missing in style.xml

```
<?xml version="1.0" encoding="utf-8"?>
<resources>

    <style name="ThemeOverlay.HRAppTheme.MaterialCalendar" parent="@style/ThemeOverlay.MaterialComponents.MaterialCalendar">
    </style>

</resources>
```

**Comment by fotospiro** [comment]
I've same crash with "catalog" app in demo app (Android 8) when apply "custom theme" and "date range".

**Comment by drchen** [comment]
It's fixed with 1.5.0-alpha01 just released last week.
